### ISSUE TITLE
Tab bar randomly disappears

### ISSUE THREAD

**Issue by ypfmde** [kovidgoyal/kitty#1563]
Using the latest version 0.13.3 on Manjaro with the xfce desktop, I recently have the following issue: When opening up a kitty terminal, then the terminal doesn't show the prompt and things typed in. Only after pressing [Enter] the previous line and current line are displayed.

That, however, is a minor thing, because after hitting [Enter] once lines are displayed correctly.

Nevertheless, this might be related to a more cumbersome issue: The tab bar randomly disappears. It often reappears when mouse-clicking into a different terminal and returning to the original terminal afterwards. However, afterwards it does not take long that the tab bar disappears again for no apparent reason.

This also happens when starting kitty with --config NONE.

**Comment by kovidgoyal** [comment]
Did you recently do a system update? Most likely this issue is a duplicate of #1484

**Comment by ypfmde** [comment]
Thanks, I had not seen #1484. Indeed, downgrading mesa fixed the issue. (And hopefully mesa 19.0.3 will be in the Manjaro repositories soon.)
